types/is-callable/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for is-callable 1.1
+// Project: https://github.com/ljharb/is-callable
+// Definitions by: Daniel <https://github.com/nieltg>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function isCallable(val?: any): val is (...args: any[]) => any;
```

Comment:
why is the val parameter optional it should be required

Reply:
Oh, I'm sorry. I forgot that `any` also includes `undefined` type.

Thank you very much.